vendor/github.com/optiopay/kafka/proto/messages.go

```diff
@@ -226,9 +226,15 @@ func ComputeCrc(m *Message, compression Compression) uint32 {
 // writeMessageSet writes a Message Set into w.
 // It returns the number of bytes written and any error.
 func writeMessageSet(w io.Writer, messages []*Message, compression Compression) (int, error) {
+	// The RECORDS type is nullable.
```

Comment:
@rlenglet it seems this files are being updated without changing the commit ID from Gopkg.toml. How's this possible? :thinking:

Reply:
Now that this commit has been merged into our optiopay/kafka fork (https://github.com/cilium/kafka/pull/3), I'm remove the direct commit from this PR and replacing with updating this with dep / `Gopkg.toml`.
Coming soon.